### PR TITLE
Update README to specify version of MongoDB

### PR DESCRIPTION
**Pull request by MiaAltieri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ Security issues in the Charmed MongoDB Operator can be reported through [LaunchP
 
 ## Contributing
 
-Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/mongodb-operator/blob/main/CONTRIBUTING.md) for developer guidance.
```
Comment (marceloneppel) [inline README.md:33]: I couldn't find the diff here. Maybe is some mistake from GIT itself.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
 
-MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB on physical or virtual machines as a single replica.
+MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB (vs 5.0) on physical or virtual machines as a single replica.
```
Comment (marceloneppel) [inline README.md:8]: Great addition, Mia!

**Review by marceloneppel — COMMENTED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (MiaAltieri) [reply README.md:33]: yeah I have _no idea_ why it did this, I think this also happened on one of my previous PRs

Comment (zmraul) [reply README.md:33]: Ah, I saw that as well on the previous PR. I thought I was going crazy :D

**Pull request merged**